src/customizations/components/theme/Blockquote/Blockquote.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Blockquote({ citation, author }) {
+  return (
+    <div className="eeaBlockquote noQuotes">
```

Comment:
Please use a better system for naming css classes. The regular CSS style is kebab-case.